feat: new introductory javascript project (#54305)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f17294346b7d69e79db3d.md

---
id: 660f17294346b7d69e79db3d
title: Step 27
challengeType: 0
dashedName: step-27
---

# --description--

> You are now ready to start building your pyramid generator. Your `character` variable will be the character that is used in the pyramid.

Reviewer: ```suggestion
You are now ready to start building your pyramid generator. Your `character` variable will serve as the building block for the pyramid.
```

Reviewer: I thought "building block" would be a more suitable. It is also a little less confusing.
